apps/src/applab/applab.js

```diff
@@ -687,6 +687,11 @@ Applab.init = function(config) {
     // and sized in drawDiv().
     Applab.setLevelHtml(level.levelHtml || level.startHtml || "");
 
+    if (!!config.level.projectTemplateLevelName) {
+      studioApp.displayAlert('#codeWorkspace', 350, 'warning',
```

Comment:
If we have to pass in size because the toolbox width varies, can we derive this parameter from the toolbox itself, or at least point at whatever thing sets the toolbox width?